stl/inc/flat_set

```diff
@@ -291,21 +287,21 @@ public:
 
     _NODISCARD container_type extract() && noexcept(is_nothrow_move_constructible_v<_Container>) /* strengthened */ {
         // always clears the container (N4950 [flat.set.modifiers]/14 and [flat.multiset.modifiers]/10)
-        _Clear_guard<_Base_flat_set, is_nothrow_move_constructible_v<_Container>> _Guard{this};
-        return _STD move(_Get_cont());
+        _Clear_guard<_Base_flat_set> _Guard{this};
+        return _STD move(_Mycont);
     }
     void replace(container_type&& _Cont) {
-        _STL_ASSERT(_Check_sorted(_Cont.cbegin(), _Cont.cend()), _Msg_not_sorted);
-        _Clear_guard<_Base_flat_set, is_nothrow_move_assignable_v<_Container>> _Guard{this};

```

Comment:
This optimization was valid, but I think we'd better put this off after solving the remaining invariant-safety issues in insert/erase methods. (And `"bool _Noexcept = false"` might not be the most suitable form.)